pyproject.toml

```diff
@@ -78,6 +78,7 @@ pygraphviz = ["pygraphviz"]
 pydot = ["pydot"]
 plotting = ["matplotlib"]
 causalml = ["causalml", "llvmlite", "cython"]
+autogluon = ["autogluon-tabular"]
```

Comment:
I think the package name is autogluon.tabular.

Reply:
They both work. According to https://peps.python.org/pep-0503/#normalized-names, a dot gets replaced by a dash. Note also that we use `autogluon-tabular` already in other places in this project file. Also tested this locally. Both names work.